mapping/README.md

```diff
@@ -147,19 +148,53 @@ If transform is not specified values will be transformed as is.
 
 A Math transformation can be defined by the `math` attribute.
 It accepts [py-expression-eval](https://github.com/AxiaCore/py-expression-eval/) formulas as argument.
-The DBC feeder expects the DBC value to be represented as `x` like in the example below.
+The DBC feeder expects the DBC value to be represented as `x`.
+
+When evaluating what transformation that is needed one must study both the DBC signal and the VSS signal. An example is given below for mirror tilt of left mirror.
+
+**DBC**
+
+The signal `VCLEFT_mirrorTiltYPosition` provide mirror tilt.
+
+```
+SG_ VCLEFT_mirrorTiltYPosition : 41|8@1+ (0.02,0) [0|5] "V"  Receiver
+```
+
+An introduction to DBC file syntax can be found [here](https://www.csselectronics.com/pages/can-dbc-file-database-intro). The specification above shows that on CAN the tilt is represented as 0-5 Volts. The value is sent with a scaling of 0.02 and uses 8 bits, i.e. 5 Volts is transmitted as 250, but that information is not needed for the can-provider transformation as the DBC scaling/offset is performed automatically by the CAN middleware.
+
+The
+
+
+**VSS**
+
+The corresponding signal in VSS use -100 percent to +100 percent as range and int8 as datatype:
+
+```
+Vehicle.Body.Mirrors.DriverSide.Tilt:
+  datatype: int8
+  unit: percent
+  min: -100
+  max: 100
+  type: actuator
+  description: Mirror tilt as a percent. 0 = Center Position. 100 = Fully Upward Position. -100 = Fully Downward Position.
+  ```
+
+With an assumptions that 5 Volts corresponds to fully upward (+100%) and 0 Volts corresponds to
+fully downward (-100%) then one could define mapping like in the example below. This is based on the assumption that the DBC signal contain actual value and we want to inject actual value to VSS.
```

Comment:
Waht does it mean? We assume, the CAN signal described by the DBC represents the current value of the Mirror Tilt? (as in "sensor"?)

Reply:
I will rephrase that. 

We have a potential logical problem here. First we do not know if the CAN signal represents wanted tilt or actual tilt. I suspect that it represents wanted tilt and that there is no signal that represents actual tilt. But when we specify `dbc2vss` that will result in that the value is injected as actual value into Databroker, we can as of today not control that. In most cases that is not a problem as the actual tilt with a small delay will correspond to wanted tilt. I will add something like

`This results in that kuksa-can-provider will take the value from the CAN signal and inject to Databroker as actual value.`

...instead.